### PR TITLE
FLUID-6288 Add Dockerfile

### PR DESCRIPTION
**Pull request by gtirloni**
This is needed so the website can be served from a container.

To test:

```
$ docker build -t floeproject .
$ docker run --rm -ti -p 8000:80 floeproject
$ open http://localhost:8000
```

### CODE REVIEW

**Comment by BlueSlug** [comment]
This built fine for me in my environment, I have no further feedback to give :)

**Comment by gtirloni** [comment]
@BlueSlug thanks! :)

File: README.md [hunk 1]
```diff
@@ -13,6 +13,21 @@ This Repository
...
+You can serve the website from a [Docker](https://docs.docker.com/get-docker) container.
+
+Once you have Docker installed, run the following commands to build a Docker image and start a container:
+
+* Build the image: `docker build -t floeproject .`
+* Run the container: `docker run --name floeproject -p 8000:80 floeproject`
+
+The website will be available at [http://localhost:8000](http://localhost:8000)
+
+If you make changes to the HTML code, repeat the steps to build the image and start a new container.
```
Comment (jobara) [inline README.md:28]: The documentation is missing the information about how to remove the docker image.

**Comment by jobara** [comment]
merged at a5984997f51d8767601e887b216ec143b47070e6

**Pull request merged**